src/less/documentation-payex.less

```diff
@@ -1,3 +1,61 @@
 @import "_variables-swedbankpay.less";
 @import "_variables-payex.less";
 @import "documentation-swedbankpay.less";
+
+body {
+    font-family: @brand-default;
+
+    p {
+        &.lead {
+            font-family: @brand-black;
+            font-weight: 500;
+        }
+    }
+}
+
+.documentation {
+    font-family: @brand-default;
```

Comment:
This font-family rule should probably be moved outside of the `.documentation` class.